certbot/renewal.py

```diff
@@ -304,6 +304,8 @@ def _renew_describe_results(config, renew_successes, renew_failures,
         notify(report(renew_skipped, "skipped"))
     if not renew_successes and not renew_failures:
         notify("No renewals were attempted.")
+        if config.pre_hook is not None or config.post_hook is not None:
```

Comment:
You're right that that one should be kind of obvious. I'm cool with not checking, or with checking.